client/src/controllers/KeyboardController.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus';
+import Mousetrap from 'mousetrap';
+
+export class KeyboardController extends Controller<HTMLButtonElement> {
+  static values = { key: String };
+
+  handleKey(event: Event) {
+    event.preventDefault();
```

Comment:
```suggestion
    if (event.preventDefault) event.preventDefault();
```

Safer to do a check here, it's possible that this callback can be fired a different way (e.g. using `Mousetrap.trigger(...` and it will not have an event that is preventable.

Reply:
Thank you! I didn't think of that 😅